universal_doc_importer/github/connectors.py

```diff
@@ -44,5 +46,14 @@ def get_repo_map(self, url, extensions):
                 mapping.addChild(file_content.path)
         return filter_by_extension(mapping.makeDict(), extensions)
 
+    def get_files(self, repo_map):
+        repo_name = list(repo_map)[0]
+        repo_name = repo_name.split('-')[0] + '/' + repo_name.split('-')[1]
```

Comment:
`'/'.join(repo_name.split('-')[:2])`

also add a comment in the code with an example, please